sources/platform/proxy/usage.md

```diff
@@ -11,24 +11,29 @@ slug: /proxy/usage
 
 ## Connection settings
 
-Below are the HTTP proxy connection settings for Apify Proxy.
+To connect to the Apify Proxy, you use the [HTTP proxy protocol](https://en.wikipedia.org/wiki/Proxy_server#Web_proxy_servers). This means that you need to configure your HTTP client to use the proxy server at `proxy.apify.com:8000` and provide it with your Apify Proxy password and the other parameters described below.
+
+The full connection string has the following format:
+
+```text
```

Comment:
First, show the connection string before diving into its components. This is explains it better